s/scala-library/scala-library_ubi8.sh

```diff
@@ -0,0 +1,58 @@
+ ----------------------------------------------------------------------------
+#
+# Package       : Scala-library
+# Version       : 2.13.6
+# Source repo   : https://github.com/scala/scala
+# Tested on     : UBI 8
+# Script License: Apache License Version 2.0
+# Maintainer    : Swati Singhal <swati.singhal@ibm.com>
+#
+# Disclaimer: This script has been tested in root mode on given
+# ==========  platform using the mentioned version of the package.
+#             It may not work as expected with newer versions of the
+#             package and/or distribution. In such case, please
+#             contact "Maintainer" of this script.
+#
+# ----------------------------------------------------------------------------
+#!/bin/bash
+
+PACKAGE_VERSION=2.13.6
+
+echo "Usage: $0 [-v <PACKAGE_VERSION>]"
+echo "       PACKAGE_VERSION is an optional paramater whose default value is 2.13.6"
+
+PACKAGE_VERSION="${1:-$PACKAGE_VERSION}"
+
+yum update -y
+yum install -y git curl
+
+#install java
+yum install -y java java-devel
+which java
+ls /usr/lib/jvm/
+export JAVA_HOME=/usr/lib/jvm/java-1.8.0-openjdk-1.8.0.312.b07-1.el8_4.ppc64le
```

Comment:
It's better to install specific version of openjdk/java. In this particular case `java-1.8.0-openjdk-devel`.

Also, assigning absolute path to `JAVA_HOME` may create issues later due to availability of a higher sub-version, causing the path in `JAVA_HOME` non-existent.

You can use [this](https://github.com/ppc64le/build-scripts/blob/master/p/pip-ray/pip-ray_0.7.7_rhel7.6.sh#L26-L28) as reference.